chain/network/src/peer_store.rs

```diff
@@ -135,11 +168,12 @@ impl PeerStore {
     }
 
     /// Return unconnected or peers with unknown status that we can try to connect to.
-    /// Peers with unknown addresses are filtered out
-    pub fn unconnected_peers(&self, ignore_list: &HashSet<PeerId>) -> Vec<PeerInfo> {
+    /// Peers with unknown addresses are filtered out.
+    pub fn unconnected_peers(&self, ignore_list: &HashSet<PeerId>, me: &PeerId) -> Vec<PeerInfo> {
```

Comment:
Why not just add `me` to ignore_list before calling this function? I think `ignore_list` means what should be ignored and `me` should rightfully be ignored.

Reply:
I was thinking in performance, that would require we add `me` to the `ignore_list`, by cloning completely the `ignore_list` or adding and removing `me` from it.